acceptance/analyzer_test.go

```diff
@@ -499,6 +575,16 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 			when("app image exists", func() {
 				it("does not restore app metadata", func() {
 					h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 restores app metadata")
+
+					analyzeFlags := []string{"-daemon"}
+					if api.MustParse(platformAPI).Compare(api.MustParse("0.7")) >= 0 {
+						analyzeFlags = append(analyzeFlags, []string{"-run-image", "some-run-image"}...)
+					}
+
+					var execArgs []string
+					execArgs = append([]string{ctrPath(analyzerPath)}, analyzeFlags...)
+					execArgs = append(execArgs, fixtures.daemonAppImage)
```

Comment:
why don't we initialize the `execArgs` in one line?

Reply:
It's so that we could splat `analyzeFlags`.